modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -96,7 +96,7 @@ public class {{classname}}{{#parent}} extends {{{parent}}}{{/parent}}{{^parent}}
     {{#openApiNullable}}
     {{^required}}
     if (this.{{name}} == null{{#isNullable}} || !this.{{name}}.isPresent(){{/isNullable}}) {
-      this.{{name}} = {{#isNullable}}JsonNullable.of({{{defaultValue}}}){{/isNullable}}{{^isNullable}}{{{defaultValue}}}{{/isNullable}};
+      this.{{name}} = {{#isNullable}}JsonNullable.of({{/isNullable}}{{{defaultValue}}}{{^defaultValue}}new {{#uniqueItems}}LinkedHashSet{{/uniqueItems}}{{^uniqueItems}}ArrayList{{/uniqueItems}}<>(){{/defaultValue}}{{#isNullable}}){{/isNullable}};
```

Comment:
Worth noting the change for `uniqueItems` here - could/should there be a unit test?

Reply:
there's a test: https://github.com/OpenAPITools/openapi-generator/pull/14703/commits/7eb5d5e373ccea95c108fc432a3b67d60c5b25bc#diff-367c850bfdb5159f550162fc5c095292dbe3f083fd7342689640e429d250003eR65

Reply:
That looks like an updated sample but not a test from the diff?

Reply:
sure, tests are always better. will do it in a separate PR with more tests.